src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```diff
@@ -58,6 +84,13 @@
     <Compile Include="$(RepoRoot)src\Shared\Components\WebAssemblyComponentMarker.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
+      <_Parameter1>Microsoft.AspNetCore.E2ETesting.TestTrimmedApps</_Parameter1>
+      <_Parameter2>$(TestTrimmedApps)</_Parameter2>
+    </AssemblyAttribute>
+  </ItemGroup>
```

Comment:
We ought to stop doing this, it's end up adding up and becoming painful. I'm ok with it for now, but we should move to a json file or similar on the folder that is much easier to change, update, etc. without having to deal with assembly attributes